conf.d/fluentd/in_dummy.conf

```diff
@@ -0,0 +1,54 @@
+
+# TODO: using in_dummy for smoking test.
+
+# <source>
+#     @type dummy
+
+#     dummy {"message": "my dummy message"}
+
+#     auto_increment_key index
+#     rate 1
+#     size 1
+#     tag dummy.source
+
+#     # @label @DEBUG
+# </source>
+
+# <match dummy.source>
+#   @type kafka2
+#   brokers kafka:9093
+
+#   <buffer topic>
+#     @type memory
+#     flush_interval 3s
+#   </buffer>
+
+#   <format>
+#     @type json
+#   </format>
+
+#   default_topic general-topic
+
+#   required_acks -1
+#   max_send_limit_bytes 10485760
+# </match>
```

Comment:
Can this be replaced with `@label @KAFKA`?

Reply:
> Can this be replaced with `@label @KAFKA`?

Right, we may uncomment them and use `@label @KAFKA` instead. Too busy to test the dummy. I originally want to use in_dummy for the smoking test.